platform/keyword-registry/src/main/java/org/codice/ddf/keyword/KeywordRegistry.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.keyword;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Class that provides keyword internalization mappings for vernacular preferences.
+ *
+ * <p><b> This code is experimental. While this interface is functional and tested, it may change or
+ * be removed in a future version of the library. </b>
+ */
+public interface KeywordRegistry {
```

Comment:
❓ Was the Java I18N support considered before creating this registry?

I could see having a `i18n` folder with resource bundle property files in the distribution which could then be extended for new languages or customer specific localization.

https://docs.oracle.com/javase/tutorial/i18n/resbundle/propfile.html
https://docs.oracle.com/javase/8/docs/api/java/util/PropertyResourceBundle.html

Reply:
This was not considered however it seems to be doing exactly what I'm doing. I could go either way on this issue. If changing it to use Java I18N will provide us with more functionality then I'm all for it otherwise the simple approach I'm currently taking might be the best way forward

Reply:
I think we should go the Java i18N route as it is the standard way for doing these things in Java.